src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/PreferInterpolationRuleTests.cs

```diff
@@ -453,5 +412,19 @@ public void SingleArgs_DontSuggestFix(string text)
         {
             ExpectPass(text);
         }
+
+    [TestMethod]
+    public void Codefix_ignores_nested_concats() => AssertCodeFix("""
+var test = conc|at('abc', concat('def', 'ghi'), 'jkl')
+""", """
+var test = 'abc${concat('def', 'ghi')}jkl'
```

Comment:
I removed the ability for this linter rule to recursively fix `concat()` - felt unnecessarily complex, unlikely to come up in the real world, and easy to handle if it does.